### PR TITLE
Remove docs, update readme

### PR DESCRIPTION
**Pull request by iamvery**
I created issues for everything in the docs. This removes them and updates readme. Good enough?

@tech256/folks

### CODE REVIEW

**Comment by chadxz** [comment]
LGTM 👍

File: readme.md [hunk 1]
```diff
@@ -1,5 +1,7 @@
-> North Alabama Tech Jobs Board
+# [tech256][https://tech256.com] Jobs Board
 
- - [Who is Hiring?](who-is-hiring.md)
- - [Who Wants to Be Hired?](who-wants-to-be-hired.md)
- - [Freelancer? Seeking Freelancer?](freelancer-seeking-freelancer.md)
+## Getting Started
+
+Check out the [open issues](https://github.com/tech256/jobs/issues) for who is hiring and who is looking to be hired.
+Listing are organized using labels.
```
Comment (chadxz) [inline readme.md]: Listing**s**

File: readme.md [hunk 2]
```diff
@@ -1,5 +1,7 @@
-> North Alabama Tech Jobs Board
+# [tech256][https://tech256.com] Jobs Board
 
- - [Who is Hiring?](who-is-hiring.md)
- - [Who Wants to Be Hired?](who-wants-to-be-hired.md)
- - [Freelancer? Seeking Freelancer?](freelancer-seeking-freelancer.md)
+## Getting Started
+
+Check out the [open issues](https://github.com/tech256/jobs/issues) for who is hiring and who is looking to be hired.
```
Comment (chadxz) [inline readme.md]: s/who is looking to be hired/available candidates

**Comment by iamvery** [comment]
thanks @chadxz ❤️

**Pull request merged**